### PR TITLE
Go add to your app.

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

File: docs/examples/add-to-your-application/go/go.sum [hunk 1]
```diff
@@ -0,0 +1,94 @@
+github.com/andybalholm/brotli v1.0.2/go.mod h1:loMXtMfwqflxFJPmdbJO0a3KNoPuLBgiu3qAvBg8x/Y=
```
Comment (gkaemmer) [inline docs/examples/add-to-your-application/go/go.sum:1]: This file seems to be gitignored in the quickstart -- do we want it here?

File: docs/examples/add-to-your-application/go/routes.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import (
+    "fmt"
+	"log"
+	"github.com/gofiber/fiber/v2"
+)
+
+func InitApp() *fiber.App {
+    oso := initOso()
+	app := fiber.New()
```
Comment (gkaemmer) [inline docs/examples/add-to-your-application/go/routes.go:11]: Seems to be some mixed tabs/spaces in this file

File: docs/examples/add-to-your-application/go/go.sum (hunk 1 above)
Comment (dhatch) [reply docs/examples/add-to-your-application/go/go.sum:1]: Nope will remove

File: docs/examples/add-to-your-application/go/models.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	Name   string
+	Repository Repository
+}
+
+type User struct {
+	Roles []Role
+}
+
+var usersDb = map[string]User{
+    "larry":  {Roles: []Role{{Name: "admin", Repository: reposDb["gmail"]}}},
```
Comment (gkaemmer) [inline docs/examples/add-to-your-application/go/models.go:30]: Any reason to exclude other the other two examples here?

**Review by gkaemmer — APPROVED** [review]
Looks good to me, just a couple comments

File: docs/examples/add-to-your-application/go/models.go (hunk 3 above)
Comment (dhatch) [reply docs/examples/add-to-your-application/go/models.go]: I have been excluding them in all the add to your apps - I guess not for any great reason, I'm not sure they add much in the context of the page though, and users won't run the add to your app code or experiment with it.

File: docs/examples/add-to-your-application/go/routes.go (hunk 2 above)
Comment (dhatch) [reply docs/examples/add-to-your-application/go/routes.go:11]: thanks ran go fmt

**Pull request merged**